### PR TITLE
:memo: Rewrites INSTALLATION.md for clarity purposes

### PR DESCRIPTION
**Pull request by AlexandreSajus**
We have had reports of users failing to contribute to Taipy GUI due to installation errors. I rewrote the INSTALLATION markdown in the hopes that it helps new contributors pass this barrier of entry.

# Before:
image

# After:
image

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -1,92 +1,91 @@
...
+**Select "Automatically install the necessary tools" when asked.**
+
+### 3 - Build the JavaScript bundle
+
+Open a new terminal and **move to the gui directory** of the repository.
+
+Then, run the following commands:
+- Install the DOM dependencies
+```bash
+cd dom
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md:63]: missing `cd gui`

File: INSTALLATION.md [hunk 2]
```diff
@@ -1,92 +1,91 @@
...
+npm i
+cd ..
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md]: should be in the next block
before `npm i --omit=optional`

File: INSTALLATION.md [hunk 3]
```diff
@@ -1,92 +1,91 @@
...
 
-You need to make sure that the [Node.js](https://nodejs.org/) JavaScript runtime version 18
-or above is installed on your machine.<br/>
-Note that Node.js comes with the [`npm` package manager](https://www.npmjs.com/) as part
-of the standard installation.
+First you need to install <a href="https://nodejs.org/" target="_blank">Node.js</a> on your system.
 
-Here is the sequence of commands that must be issued, assuming your current directory
-is the root directory of the repository:
+**Select the "Recommended For Most Users" version, and follow the instructions for your system.**
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md:52]: latest LTS

File: INSTALLATION.md [hunk 4]
```diff
@@ -1,92 +1,91 @@
...
-This creates the directory `src/taipy/gui/webapp` in the root directory of the repository
+After a few minutes, this creates the directory `src/taipy/gui/webapp` in the root directory of the repository
 where the front-end code for Taipy GUI is split into a set of JavaScript bundles.
 
-### Debugging the JavaScript bundle
+### 4 - Install the package as editable
+
+In a terminal, **at the root of the repository**, run:
+```bash	
+pip install -e . --user
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md:83]: Is this how you work with dev version ?

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

File: INSTALLATION.md (hunk 1 above)
Comment (FabienLelaquais) [reply INSTALLATION.md:63]: Yeah,,, this appears in the text above, but it would be far more clear.

File: INSTALLATION.md (hunk 3 above)
Comment (AlexandreSajus) [reply INSTALLATION.md:52]: image
Latest LTS is less clear than my statement:

File: INSTALLATION.md (hunk 4 above)
Comment (AlexandreSajus) [reply INSTALLATION.md:83]: Yes, this is how I managed to install it. Is there something else I should add?

File: INSTALLATION.md (hunk 2 above)
Comment (AlexandreSajus) [reply INSTALLATION.md]: What do you mean?

Comment (FredLL-Avaiga) [reply INSTALLATION.md]: move cd .. to the next block of code

File: INSTALLATION.md (hunk 4 above)
Comment (FredLL-Avaiga) [reply INSTALLATION.md:83]: no all good

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
👍

File: INSTALLATION.md (hunk 2 above)
Comment (AlexandreSajus) [reply INSTALLATION.md]: Ah yes, done!

**Pull request merged**